apps/web/app/Http/Controllers/Auth/AuthenticatedSessionController.php

```
@@ -35,7 +37,24 @@ public function store(LoginRequest $request): RedirectResponse
+            $invitationController = new WorkspaceInvitationController;
+            $accepted = $invitationController->acceptInvitationByToken($invitationToken, $user);
+
+            if ($accepted) {
+                return redirect()->intended(route('dashboard', absolute: false))
+                    ->with('success', 'Successfully joined the workspace!');
+            } else {
+                // If invitation acceptance fails, still proceed with normal login
+                // but show a warning message
+                return redirect()->intended(route('dashboard', absolute: false))
+                    ->with('warning', 'Could not join workspace. The invitation may be invalid or expired.');
+            }
```

PHPMD (2.15.0)

The method store uses an else expression. Else clauses are basically not necessary and you can simplify the code by not using them. (Clean Code Rules)

(ElseExpression)